releasenotes/notes/remove_0.15_qiskit_scheduler_utils-fab5ec8a7ceddb56.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The functions ``qiskit.scheduler.utils.format_meas_map``,  ``qiskit.scheduler.utils.measure``, and
+    ``qiskit.scheduler.utils.measure_all``
+    had been moved to :func:`qiskit.pulse.utils.format_meas_map`, :func:`qiskit.pulse.macros.measure`,
+    and :func:`qiskit.pulse.macros.measure_all` respectively. The previous location was deprecated
+    in Qiskit 0.20.0 (Terra 0.15.0, released on 2020-08-10) and it is not longer supported.
```

Comment:
I'm not certain that the cross-references here will resolve.  It's not clear to me whether those functions are actually meant to be used directly or documented - seems to me a little more that they're only meant to be called via the pulse builder interface, such as the wrapper `pulse.builder.measure_all`?

(Also, "not longer supported" -> "no longer supported".)

Reply:
cross reference (and typo) removed in 55b3b7ed3d0eede0269e7cef05394ccfa6ccc81f